tests/end2end-tests/graphstorm-mt/mgpu_test.sh

```diff
@@ -190,3 +190,18 @@ then
     echo "The number of save models $cnt is not equal to the specified topk 3"
     exit -1
 fi
+
+echo "**************[Multi-task gen embedding] dataset: Movielens, RGCN layer 1, node feat: fixed HF BERT, BERT nodes: movie, load from saved model"
+python3 -m graphstorm.run.gs_gen_node_embedding --workspace $GS_HOME/training_scripts/gsgnn_mt/ --num-trainers $NUM_TRAINERS --use-mini-batch-infer false --num-servers 1 --num-samplers 0 --part-config /data/movielen_100k_multi_task_train_val_1p_4t/movie-lens-100k.json --ip-config ip_list.txt --ssh-port 2222 --cf ml_nc_ec_er_lp.yaml --save-embed-path /data/gsgnn_mt/save-emb/ --restore-model-path /data/gsgnn_mt/epoch-2/ --restore-model-layers embed,gnn --logging-file /tmp/train_log.txt --logging-level debug --preserve-input True
+
+error_and_exit $?
+
+cnt=$(ls -l /data/gsgnn_mt/save-emb/ | wc -l)
+cnt=$[cnt - 1]
+if test $cnt != 2
+then
+    echo "The number of saved embs $cnt is not equal to 2 (for movie and user)."
+fi
+
+# Multi-task will save node embeddings of all the nodes.
+python3 $GS_HOME/tests/end2end-tests/check_infer.py --train-embout /data/gsgnn_mt/emb/ --infer-embout /data/gsgnn_mt/save-emb/ --link-prediction
```

Comment:
Do we need to check if the stored infer_ntypes are sorted?

Reply:
No. As sorting is done by the remap script.
Here we only test if the embeddings saved in the training process is same as the embeddings saved in gen_embedding are same.